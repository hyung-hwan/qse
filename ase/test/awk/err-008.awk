
BEGIN {
	local substr;
	substr = 20;
}

