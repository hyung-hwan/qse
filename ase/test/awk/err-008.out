PARSE ERROR: CODE [56] LINE [3] built-in function 'substr' redefined
